README.md

```diff
@@ -14,8 +14,13 @@ composer require lmc/aerospike-cache
 ```
 
 ## Usage
+```php
+$aerospike = new \Aerospike(['hosts'=>[['addr'=>'127.0.0.1', 'port'=>3000]]]);
+$aerospikeCache = new AerospikeCache($aerospike, 'aerospkeNamespace');
+```
 
-TODO usage
+AerospikeCache uses PSR-6 caching interface for manipulation with the data.  
+The AerospikeSymfony cache also comes bundeled with Symfony Cache adapter for DoctrineCache
```

Comment:
What is `AerospikeSymfony`? Should it be `AerospikeCache` ?

Reply:
Yes thanks for noticing